migrate/migrations/20250311113136-track-rotation-updates.sql

```diff
@@ -0,0 +1,67 @@
+-- +migrate Up
+CREATE TABLE entity_updates(
+    id bigserial PRIMARY KEY,
+    entity_type text NOT NULL,
+    entity_id uuid NOT NULL,
+    created_at timestamptz NOT NULL DEFAULT now()
+)
+WITH (
+    autovacuum_vacuum_threshold = 50, -- Lower threshold to trigger vacuum sooner
```

Comment:
question: the only other reference to this also says it's using a lower threshold, but the value is set to `5`. what are we lowering these from?

Reply:
We're actually not lowering the threshold itself from Postgres’s default (which is 50 rows by default). We're explicitly setting it here just for clarity. The meaningful change we're introducing is reducing autovacuum_vacuum_scale_factor from its default of 0.2 (20%) to 0.05 (5%), causing vacuum operations to trigger sooner relative to table size.

Both values (autovacuum_vacuum_threshold and autovacuum_vacuum_scale_factor) are explicitly defined here just for clarity and readability, even though only the scale factor is actually changing from the default behavior.